ocfweb/templates/404.html

```diff
@@ -14,6 +14,6 @@
         height: 1000px;
     }
 </style>
```

Comment:
My previous comment still stands: stylesheets should be specified in the SCSS directory, and not in the middle of a `<div>`.

Reply:
Will add the stylesheet improvements later!

Reply:
I do think this would be good to do, however currently it is a bit harder to do this since some kind of identifying class needs to be added to the body of the email pages that isn't currently added, so some of the Django side of things needs to be changed to accomplish that.